provision-osx/tasks/configure_buildfarm_proxy.yml

```diff
@@ -26,6 +26,22 @@
     - proxy_user is not defined or proxy_user == ''
     - proxy_pass is not defined or proxy_pass == ''
 
+- pause:
```

Comment:
if we add a register to the command above we can only show this prompt when necessary.
```
-
  name: Set node proxy configuration with authentication
  ...
  register: proxy_config_output
  ...

- 
  name: Is proxy config sucessful
  set_fact:
       proxy_config_failed: true
  when:  '25308' in proxy_config_output.stderr
```

currently the prompt shows whether it was successful or not.